File: Taller/Taller/Models/Cliente.cs
namespace TallerMecanico.Models
{
    public class Cliente
    {
        public int Id { get; set; }
        public required string Nombre { get; set; } = string.Empty;
        public required string Telefono { get; set; } = string.Empty;
        public required string Direccion { get; set; } = string.Empty;
        public required string Correo { get; set; } = string.Empty;
        public DateTime FechaRegistro { get; set; } = DateTime.Now;

    }
}

File: java/TallerJava/src/main/java/Modelos/Cliente.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Modelos;

/**
 *
 * @author Administrador
 */

public class Cliente {
    private int id;
    private String nombre;
    private String telefono;
    private String direccion;
    private String correo;
    private String fechaRegistro;

    // Getters
    public int getId() { return id; }
    public String getNombre() { return nombre; }
    public String getTelefono() { return telefono; }
    public String getDireccion() { return direccion; }
    public String getCorreo() { return correo; }
    public String getFechaRegistro() { return fechaRegistro; }

    // Setters
    public void setId(int id) { this.id = id; }
    public void setNombre(String nombre) { this.nombre = nombre; }
    public void setTelefono(String telefono) { this.telefono = telefono; }
    public void setDireccion(String direccion) { this.direccion = direccion; }
    public void setCorreo(String correo) { this.correo = correo; }
    public void setFechaRegistro(String fechaRegistro) { this.fechaRegistro = fechaRegistro; }
}